### DISCUSSION TITLE
storage/hwameistor/intro/scheduler/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1670] [Announcements]
# storage/hwameistor/intro/scheduler/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/intro/scheduler/

**Comment by liyuerich** [comment]
kubectl apply -f deploy/scheduler.yaml
error: the path "deploy/scheduler.yaml" does not exist

**Comment by WANG0608GitHub** [comment]
Fixed in PR #5285
